GetStream/stream-chat-android
# CAS-942 Add docs/UI/setup section

https://stream-io.atlassian.net/browse/CAS-942

### 🧪 Testing

1. Go to documentation folder
2. Run `yarn run start`
3. Check out `UI > Setup` section

### ☑️ Checklist
